File: leetCode/fukun/201-300/lc_226.js
var invertTree = function (root) {
    const helper = function (node) {
        if (!node) return;
        if (node.left || node.right) [node.left, node.right] = [node.right, node.left];
        helper(node.left);
        helper(node.right);
    };
    helper(root);
    return root;
};

File: leetCode/XSS/lc_226.java
public class lc_226 {
    public static void main(String[] args) {

    }
    static class Solution {
        public TreeNode invertTree(TreeNode root) {
            if(root==null)return root;
            TreeNode temp=root.right;
            root.right=root.left;
            root.left=temp;
            invertTree(root.left);
            invertTree(root.right);
            return root;
        }
    }
}
